src/handlebars/getinvolved.handlebars

```diff
@@ -21,25 +21,23 @@
   </div>
 
   <div class="margin-bottom">
-    <h2 class="bold">I'm an OpenJDK Developer</h2>
+    <h2 class="bold">I'm an OpenJDK and/or an Eclipse OpenJ9 Developer</h2>
     <p>
       We provide <a target="_blank" href="https://www.github.com/AdoptOpenJDK">infrastructure as code</a>,
-      and a <a href="https://ci.adoptopenjdk.net">Build and test farm</a> for OpenJDK developers.
-      A future version of this build farm will act as a patch submission mechanism for OpenJDK developers to submit, review and test patches for
-      multiple platforms <strong>before</strong> submitting to the <a href="https://openjdk.java.net">OpenJDK project</a>.
+      and a <a href="https://ci.adoptopenjdk.net">Build and test farm</a> for OpenJDK and Eclipse OpenJ9 developers.
+      A future version of this build farm will act as a patch submission mechanism for OpenJDK and Eclipse OpenJ9 developers to submit,
+      review and test patches for multiple platforms <strong>before</strong> submitting to their respective upstream project.
     </p>
 
     <h3 class="bold">Source Forests</h3>
     <p>
-      We've started with jdk8u and jdk11u with test coverage from the open jtreg tests, providing jcov test coverage
-      reports. Later on other popular forests such as valhalla etc will be added.
+      <strong>OpenJDK:</strong> We have clones starting from OpenJDK 8 (jdk8u) all the way through to the latest development version.
+      We also cover some projects that have yet to be merged with mainline, e.g. Projects Amber, Valhalla and more.
     </p>
-
     <p>
-      There may be some areas of overlap with other past/present efforts, but that's OK.  We expect any duplication to
-      converge in due course and have started working with other platform specific OpenJDK build efforts.
+      <strong>Eclipse OpenJ9:</strong> We pull directly from <a href="https://github.com/eclipse/openj9">Eclipse OpenJ9's GitHub</a>. OpenjDK class libraries are
```

Comment:
The "J" in "OpenjDK" isn't capitalized.
(Also, Eclipse OpenJ9 doesn't have its own GitHub. It has a GitHub repository.)

Reply:
Fixed, thank you!